### PR TITLE
feat: Clean up releases common content

### PR DESCRIPTION
**Pull request by dcramer**
- Remove duplicated releases pages
- Add Ruby example
- Add UE4 example
- Add notateUnsupported to PlatformContent (on by default)

### CODE REVIEW

**Comment by dcramer** [comment]
image

File: src/includes/set-release/dotnet.mdx [hunk 1]
```diff
@@ -3,3 +3,17 @@ using Sentry;
 
 SentrySdk.Init(o => o.Release = "my-project-name@2.3.12");
 ```
+
+The SDK attempts to locate the release to add that to the events sent to Sentry.
```
Comment (PeloWriter) [inline src/includes/set-release/dotnet.mdx:7]:
```suggestion
The SDK attempts to locate the release to add to events sent to Sentry.
```

File: src/includes/set-release/dotnet.mdx [hunk 2]
```diff
@@ -3,3 +3,17 @@ using Sentry;
...
+
+<Note><markdown>
+
+[Default values like 1.0 or 1.0.0.0 are ignored](https://github.com/getsentry/sentry-dotnet/blob/dbb5a3af054d0ca6f801de37fb7db3632ca2c65a/src/Sentry/Internal/ApplicationVersionLocator.cs#L14-L21).
+
+</markdown></Note>
+
+The SDK will firstly look at the [entry assembly's](<https://msdn.microsoft.com/en-us/library/system.reflection.assembly.getentryassembly(v=vs.110).aspx>) `AssemblyInformationalVersionAttribute`, which accepts a string as
```
Comment (PeloWriter) [inline src/includes/set-release/dotnet.mdx:15]:
```suggestion
The SDK will first look at the [entry assembly's](<https://msdn.microsoft.com/en-us/library/system.reflection.assembly.getentryassembly(v=vs.110).aspx>) `AssemblyInformationalVersionAttribute`, which accepts a string as
```

**Review by PeloWriter — APPROVED** [review]
thank you.

**Pull request merged**